packages/fxa-auth-server/docs/api.md

```diff
@@ -76,6 +76,12 @@ see [`mozilla/fxa-js-client`](https://github.com/mozilla/fxa-js-client).
   * [Sms](#sms)
     * [POST /sms (:lock: sessionToken)](#post-sms)
     * [GET /sms/status (:lock: sessionToken)](#get-smsstatus)
+  * [Subscriptions](#subscriptions)
+    * [GET /oauth/subscriptions/plans (:lock: oauthToken)](#get-subscriptionsplans)
+    * [GET /oauth/subscriptions/active (:lock: oauthToken)](#get-subscriptionsactive)
+    * [POST /oauth/subscriptions/active (:lock: oauthToken)](#post-subscriptionsactive)
+    * [DELETE /oauth/subscriptions/active/{subscriptionId} (:lock: oauthToken)](#delete-subscriptionsactivesubscriptionid)
+    * [POST /oauth/subscriptions/updatePayment (:lock: oauthToken)](#post-subscriptionsupdatepayment)
```

Comment:
API design nit: it's not obvious to me why these live under `/oauth/subscriptions` rather than, say, `/account/subscriptions` or even `/subscriptions`.  My perspective is that the routes under `/oauth` are specifically for *managing oauth state* such as tokens and codes, which I'm not sure this is.  But anyway, no big deal if this structure makes sense to you, just something I noticed.

Reply:
I suggested this, I think for API clarity it makes sense to me for everything authorized by an OAuth token to be under the /oauth path.

Reply:
Yeah, this was a change per review feedback from last week: https://github.com/mozilla/fxa/pull/819#pullrequestreview-227656302